buildbot.mariadb.org/utils.py

```diff
@@ -36,7 +36,7 @@ def getScript(scriptname):
       name=f"fetch_{scriptname}",
       command=["bash", "-xc", f"""
   for script in bash_lib.sh {scriptname}; do
-    wget "https://raw.githubusercontent.com/MariaDB/mariadb.org-tools/master/buildbot.mariadb.org/scripts/$script"
+    [[ ! -f $script ]] && wget "https://raw.githubusercontent.com/MariaDB/mariadb.org-tools/master/buildbot.mariadb.org/scripts/$script"
```

Comment:
This is leaving the update of these scripts to be a manual process.

While using a `curl --etag-compare`  / `--etag-save` like the aix builder would be ideal, its a fairly new version of curl that has that feature (7.68.0).

The other automated way is to fall back to github API and make use of its commit as a effective etag.

`curl -H "Accept: application/vnd.github.v3+json"  'https://api.github.com/repos/MariaDB/mariadb.org-tools/commits?path=/buildbot.mariadb.org/scripts/bash_lib.sh&per_page=1' | jq '.[].sha'`

Reply:
I think that what this achieve is not clear. Since we pull the bash library anytime we pull a script with `getScript`, on some test we are calling this function twice (for instance https://buildbot.mariadb.org/#/builders/367/builds/293) and so `bash_lib.sh` was fetched twice...

Reply:
Also we made the move to use wget (that has less deps) on workers but maybe that was not a smart move...

Reply:
if we update the scripts on github, how is the builder updated to use this script?

Reply:
@grooverdan the VM is always clean, so the script is fetched at least once for each build. The fix should account for cases where the script is fetched twice for the same build and only fetch it once. However I am not sure if it has implications for other builds where there might be things that are stored. As far as I know, all builds start with a clean env, so I don't think there is any issue.

Reply:
Only issue detected yesterday is that GitHub is doing some CDN cache and then the new version of script is not fetched immediately from GH... But that's a case that should not impact to much as those scripts will not be updated often (hopefully).